src/tokenTransferBuilders.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { assert } from "chai";
+import { Address } from "./address";
+import { TokenPayment } from "./tokenPayment";
+import { ESDTNFTTransferPayloadBuilder, ESDTTransferPayloadBuilder, MultiESDTNFTTransferPayloadBuilder } from "./tokenTransferBuilders";
+
+describe("test token transfer builders", () => {
+    it("should work with ESDT transfers", () => {
+        let payment = TokenPayment.fungibleFromAmount("COUNTER-8b028f", "100.00", 0);
+        let payload = new ESDTTransferPayloadBuilder().setPayment(payment).build();
+        assert.equal(payload.toString(), "ESDTTransfer@434f554e5445522d386230323866@64");
+    });
+
+    it("should work with ESDTNFT transfers", () => {
+        let payment = TokenPayment.nonFungible("ERDJS-38f249", 1);
+        let payload = new ESDTNFTTransferPayloadBuilder()
```

Comment:
can we add a test for semi fungible tokens as well?

Reply:
Added.